test/SpokePool.Relay.ts

```diff
@@ -1,24 +1,6 @@
-import { expect } from "chai";
-import { ethers } from "hardhat";
-import { Contract } from "ethers";
-import { SignerWithAddress, seedWallet, toWei, toBN } from "./utils";
+import { expect, Contract, ethers, SignerWithAddress, seedWallet, toWei, toBN } from "./utils";
 import { spokePoolFixture, enableRoutes, getRelayHash, modifyRelayHelper } from "./SpokePool.Fixture";
-import {
```

Comment:
@nicholaspai this might be controversial but I felt this was cleaner.

Reply:
i like it